llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -89,6 +89,7 @@ cl::OptionCategory PostLinkCat{"sycl-post-link options"};
 // clang/lib/Driver/Driver.cpp, sycl-post-link.cpp, ClangOffloadWrapper.cpp
 constexpr char COL_CODE[] = "Code";
 constexpr char COL_SYM[] = "Symbols";
+constexpr char COL_OPTS[] = "Options";
```

Comment:
Could you please clarify how this 4th file is going to be used?

Reply:
I have reworked the design and am piggybacking onto the prop file instead of adding a new file.